WalletWasabi.Documentation/WasabiSetupRegtest.md

```diff
@@ -9,9 +9,11 @@ You do not need to download the blockchain for this setup!
 
 Bitcoin Knots is working very similarly to Bitcoin Core. You can get a grasp with [this guide](https://bitcoin.org/en/developer-examples).
 
+WalletWasabi RegTest currently only works with Knots 21.2(LTS).
```

Comment:
Is this true? For mainnet and testnet we use 23.0 on backend servers currently, regtest should work with that too.

Reply:
ok, then I'll ask for some time and then I'll correct the wording as well.

Reply:
We should use the same on RegTest as the client which is 21.2. I am sharing the same data folder when using RegTest and Wasabi full node integration.

Reply:
I reopened #8825 to upgrade to 23.0

Reply:
I reopened #8825 to upgrade to 23.0 if needed.